src/contributing-guidelines.njk

```diff
@@ -194,18 +194,37 @@ templateClass: template-generic
 		<li>Promote the post on our <a href="https://twitter.com/A11YProject">Twitter</a> and <a href="https://www.linkedin.com/company/the-a11y-project/">LinkedIn</a> social media accounts.
 		<li>Feature you in <a href="{{ '/newsletter/' | url }}">our weekly newsletter</a>.</li>
 	</ul>
+	<h3 id="evaluation" class="c-heading-medium">
+		Evaluation
+	</h3>
+	<p>
+		Acceptance of content suggestions are weighed on the quality and availability of the actual content, the accessibility of the site it is hosted on, the organization&#39;s status in the industry, and the organization&#39;s mission.
+	</p>
```

Comment:
```suggestion
	<p>
		When we evaluate suggested content, we consider the following things:</p>
	<ul>
		<li>The quality of the content itself,</li>
		<li>The accessibility of the site that hosts the content,</li>
		<li>The content creator&lsquo;s standing within the broader web-accessibility community,</li>
		<li>The content creator&lsquo;s mission</li>
	</ul>
```

🖊️ Trying to avoid passive voice, and make the oevrall level of diction a little friendlier.

Reply:
Love it.